.circleci/config.yml

```diff
@@ -133,7 +133,37 @@ jobs:
             -
                 run:
                     name: Test MySQL database schema diff
-                    command: mysqldiff --skip-table-options --server1=root:@127.0.0.1:3306 --server2=root:@127.0.0.1:3306 pipeline_migrations:pipeline_automigrate
+                    command: |
+                        mysqldump \
+                            --host 127.0.0.1 \
+                            --password= \
+                            --port 3306 \
+                            --user "root" \
+                            \
+                            --no-data \
+                            --result-file "pipeline_migrations.sql" \
+                            --verbose \
+                            pipeline_migrations
+                        mysqldump \
+                            --host 127.0.0.1 \
+                            --password= \
+                            --port 3306 \
+                            --user "root" \
+                            \
+                            --no-data \
+                            --result-file "pipeline_automigrate.sql" \
+                            --verbose \
+                            pipeline_automigrate
+                        diff \
```

Comment:
This won't skip table options

Reply:
I'm in the experiment phase and there is a flag for that in mysqldump for exactly that, I just didn't want to start with enabling skips we might not need.

Reply:
Fixed in https://github.com/banzaicloud/pipeline/compare/eadf636cf60cfb95524e7d6500832b54c0743096..cff5a2ae422cd4afb400a4d4feffe80a7dfe9f6e.